tools/VersionCop/Program.cs

```diff
@@ -419,8 +419,9 @@ class Program
 
                             Console.WriteLine();
                             Console.WriteLine("*****************************************************************");
-                            Console.WriteLine($"Found in nuspec, but declared dependency version doesn't match!");
-                            Console.WriteLine($"expecting '{packageVersion}' but found '{idFoundInNuspec}'");
+                            Console.WriteLine($"DEPENDENCY VERSION MISMATCH!");
+                            Console.WriteLine($"The nfproj file has: '{packageVersion}',");
```

Comment:
This one should be the oposite: from the nuspec to the nfproj file.
The focus is on the nuspec, otherwise it could not be that obvious what is wrong.
Long version: "hey! this version was found on nuspec, which is great, buuuut the version it has declared there VVVVV is not the expected one YYYYYY, which, by then way is the one declared in the nfproj".

Now feel free to put your Shakespeare hat on an rewrite it. 😛

Reply:
That is why I changed the text to remove that assumption... it should now just state what is wrong?!

Reply:
i.e.

WAS:
```
Console.WriteLine($"Found in nuspec, but declared dependency version doesn't match!");
Console.WriteLine($"expecting '{packageVersion}' but found '{idFoundInNuspec}'");
```

NOW:
```
Console.WriteLine($"DEPENDENCY VERSION MISMATCH!");
Console.WriteLine($"The nfproj file has: '{packageVersion}',");
Console.WriteLine($"but the nuspec file has: '{idFoundInNuspec}'");
```

I could swap the console writelines around if required?!

Reply:
I was reviewing this on my phone and overlooked the last line. Apologies.
Yes, the message is clear.